content/metadata.yaml

```diff
@@ -44,6 +44,16 @@ author_info:
     affiliations:
       - Department of Systems Pharmacology and Translational Therapeutics, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
     funders:
-      - Pfizer Worldwide Research and Development
+      - Pfizer Worldwide Research, Development, and Medical
       - the Gordon and Betty Moore Foundation (GBMF4552)
       - the National Institutes of Health (R01 HG010067)
+
+-
```

Comment:
Small change, but this line needs to be indented by two spaces or the build fails